fabric-item-api-v1/src/main/java/net/fabricmc/fabric/impl/item/ShearsItemPredicate.java

```diff
@@ -0,0 +1,26 @@
+package net.fabricmc.fabric.impl.item;
+
+import com.mojang.serialization.Codec;
+
+import net.fabricmc.fabric.api.item.v1.CustomItemPredicate;
+
+import net.minecraft.item.ItemStack;
+
+public class ShearsItemPredicate implements CustomItemPredicate {
+	public static final ShearsItemPredicate INSTANCE = new ShearsItemPredicate();
+
+	private final Codec<ShearsItemPredicate> CODEC = Codec.unit(this);
```

Comment:
It should probably be a boolean codec, that way you can check if the item is not a shears.

Reply:
I've already done that, but I can't push the changes I've made, as GitHub on my laptop doesn't work in China and my VPN is terrible.